src/shell/lisa_shell

```diff
@@ -335,24 +352,37 @@ while [[ $# -gt 0 ]]; do
                         return 0
                         ;;
                 -n|--iterations)
-                        iterations="$2"
+                        LISA_TEST_ITERATIONS="$2"
+                        shift
+                        ;;
+
+                -t|--target-conf)
+                        LISA_TARGET_CONF="$2"
+                        shift
+                        ;;
+                -o|--results-dir)
+                        LISA_RESULTS_DIR="$2"
```

Comment:
As suggested above... wouldn't be nice to have default values for all the above variables?

Reply:
Since they are read using os.getenv in the Python scripts, you can assume they default to an empty string.

* LISA_TEST_ITERATIONS: the behavior has not changed
* LISA_TARGET_CONF: The default value is set by Python code. Maybe we should push these defaults to lisa_shell when possible and assume the environment variables are set in the Python code. 
* LISA_RESULTS_DIR: discussed above